content/en/docs/reference/node/kernel-version-requirements.md

```diff
@@ -0,0 +1,84 @@
+---
+content_type: "reference"
+title: Linux Kernel Version Requirements
+weight: 10
+---
+
+{{% thirdparty-content %}}
+
+Many features rely on specific kernel functionalities and have minimum kernel version requirements.
+However, relying solely on kernel version numbers may not be sufficient
+for certain operating system distributions,
+as kernel vendors such as RHEL, Ubuntu and SUSE often backport features to older kernels.
+
+## Pod sysctls {#requirements-pod-sysctl}
+
+On Linux, the `sysctl()` system call configures kernel parameters at run time. There is a command
+line tool named `sysctl` that you can use to configure these parameters, and many are exposed via
+the `proc` filesystem.
+
+Some sysctls are only available if you have a modern enough kernel.
+
+The following sysctls are supported in the [safe set](/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls)
+which has a minimal kernel version requirement:
+
+<!--
+Code: https://github.com/kubernetes/kubernetes/blob/00236ae0d73d2455a2470469ed1005674f8ed61f/pkg/util/kernel/constants.go#L19-L45
+-->
+- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27, needs kernel 3.16+);
+- `net.ipv4.tcp_keepalive_time` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_fin_timeout` (since Kubernetes 1.29, needs kernel 4.6+);
+- `net.ipv4.tcp_keepalive_intvl` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_keepalive_probes` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_syncookies` (namespaced since kernel 4.6+).
+- `net.ipv4.vs.conn_reuse_mode` (used in `ipvs` proxy mode, needs kernel 4.1+);
+
+### kube proxy `nftables` proxy mode
+
+<!--
+Code: https://github.com/kubernetes/kubernetes/blob/00236ae0d73d2455a2470469ed1005674f8ed61f/pkg/util/kernel/constants.go#L53-L56
+-->
+For Kubernetes {{< skew currentVersion >}}, the
+[`nftables` mode](/docs/reference/networking/virtual-ips/#proxy-mode-nftables) of kube-proxy requires
+version 1.0.1 or later
+of the nft command-line tool, as well as kernel 5.13 or later. (For testing/development
```

Comment:
The interface with nftables is deliberately via `nft`, because the netlink interface is poorly documented with no nice-to-use Go client code.

Maybe a future release of `kube-proxy` will switch to using netlink here.

Reply:
/cc @danwinship 
to confirm